index.html

```diff
@@ -15,13 +15,66 @@
+    <title>Afro-Game Developer Conference - Home</title>
   </head>
   <body>
+    <div id="modal-sidebar" class="hidden">
+        <div class="modal-content">
+        <i class="fa fa-times-circle" aria-hidden="true"></i>
+            <ul class="mobile-menu-list">
+                <li class="mobile-menu-list-item"><a href="index.html"><i class="fa fa-home menu-icon" aria-hidden="true"></i>Home</a></li>
+                <li class="mobile-menu-list-item"><a href="about.html"><i class="fa fa-info-circle menu-icon" aria-hidden="true"></i>About</a></li>
+                <li class="mobile-menu-list-item"><a href="#"><i class="fa fa-list-ul menu-icon" aria-hidden="true"></i>Program</a></li>
+                <li class="mobile-menu-list-item"><a href="#"><i class="fa fa-comments menu-icon" aria-hidden="true"></i>Speakers</a></li>
+                
+            </ul>
+        </div>
+    </div>
+    <header class="desktop-header">
+
+        <nav class="desktop-top-nav">
+            <ul class="top-nav-list">
+                <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+                <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                <li>English</li>
+                <li>Logout</li>
+            </ul>
+        </nav>
+        <nav class="desktop-secondary-nav">
+            <ul class="secondary-nav-list">
+                <li><img class="logo-top" src="./images/afro-games-logo.jpg" alt="afro games logo"/></li>
+                <li class="highlight">Home</li>
+                <li><a href="about.html" class="link">About</a></li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li class="borderize">Afro Dev Campaign</li>
+            </ul>
+        </nav>
+        <div class="desktop-home-background">
+            <div class="desktop-overlay">
+                <div class="conf-info">
+                    <h3 class="hello-text">"Hello! <em>AfroGameDevs</em>"</h3>
+                    <h1 class="main-title">AFRO GAME<br />DEVELOPER CONFERNECE<br />2022</h1>
+                    <p class="subtitle">
+                        A great moment for African Developers to showcase their talents in the Game Development industry. 
+                        This year's event will be hosted in Ghana.
+                    </p>
+                    <h3 class="date-time">
+                        2022.05.13(FRI) ~ 14(SAT)
+                    </h3>
+                    <p class="venue">
+                        @ Accra Conference Center 
+                    </p>
+                </div>
+            </div>
+        </div>
+    </header>
+
     <header class="home-grey-background">
         <div class="white-overlay">
             <div class="home-page-content">
-                <i class="fa fa-bars" aria-hidden="true"></i>
+                <button class="mobile-sidebar-toggle" type="button"><i class="fa fa-bars" aria-hidden="true"></i></button>
```

Comment:
- [ ] Kindly add `margin-top: 10px;` to this button, to match the original design.

Original design
![image](https://user-images.githubusercontent.com/35535130/158818667-3683c9d9-0a1e-4887-99cc-90e742205b4e.png)

Your design
![image](https://user-images.githubusercontent.com/35535130/158818756-c4e4db12-c6bb-4a2e-8792-dbdc2af23397.png)